src/main/java/org/iitr/muzo/models/Playlist.java

```diff
@@ -0,0 +1,27 @@
+package org.iitr.muzo.models;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "Playlist")
+public class Playlist implements Serializable {
```

Comment:
Please add getter setter in this.

Reply:
sure, next PR mein daal dunga .. issko accept kar dena :P